build/openwisp_openvpn/Dockerfile

```diff
@@ -0,0 +1,19 @@
+FROM kylemanna/openvpn:latest
+
+RUN apk add --no-cache postgresql-client supervisor && \
```

Comment:
why do we need `posgresql-client`?

Reply:
I am using `psql` command in this container and that is asking for postgresql-client.
The command I am using is:
```
psql -qAtc "SELECT id,key FROM config_vpn where name='${VPN_NAME}';"
```
This gets me the `UUID` and `KEY` of the vpn which I use for fetching files from API as:
```
wget -qO vpn.tar.gz --no-check-certificate \
    ${DOMAIN}/controller/vpn/download-config/$UUID/?key=$KEY
```

Reply:
I see. This is related to the comment above. We may need to think about a better solution for the default VPN in the future. For the moment let's keep it as is and let's go ahead.